### PR TITLE
init project

### PR DESCRIPTION
**Pull request by mohammed-allou**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,19 @@
 {
-  "name": "kitchen_backend",
+  "name": "kitchen-backend",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "serveur.js",
+  "dependencies": {
+    "ejs": "^3.1.5",
```
Comment (aallou) [inline package.json:7]: pourquoi?

File: serveur.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+console.log('salam');
```
Comment (aallou) [inline serveur.js:1]: A supprimer?

File: serveur.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+const express = require('express')
+const app = express()
+
+app.set('view engine', 'ejs')
+
+app.get('/', function (req, res) {
+    res.render('index', { pseudo: 'HAFID' })
```
Comment (aallou) [inline serveur.js:8]: c quoi ça?

File: views/index.ejs [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```
Comment (aallou) [inline views/index.ejs:1]: wachta hada?

File: serveur.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+})
+
+app.listen(9090, function () {
+    console.log('le port 9090 est en ecoute ! ');
```
Comment (aallou) [inline serveur.js:12]: English please

File: serveur.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
+console.log('salam');
+const express = require('express')
+const app = express()
+
+app.set('view engine', 'ejs')
```
Comment (aallou) [inline serveur.js:5]: A supprimer

File: package.json (hunk 1 above)
Comment (mohammed-allou) [reply package.json:7]: ok

File: serveur.js (hunk 3 above)
Comment (mohammed-allou) [reply serveur.js:8]: ok

File: serveur.js (hunk 5 above)
Comment (mohammed-allou) [reply serveur.js:12]: ok

File: serveur.js (hunk 2 above)
Comment (mohammed-allou) [reply serveur.js:1]: ok

File: views/index.ejs (hunk 4 above)
Comment (mohammed-allou) [reply views/index.ejs:1]: ok

File: serveur.js (hunk 6 above)
Comment (mohammed-allou) [reply serveur.js:5]: ok

**Pull request merged**